### PR TITLE
Line mark

### PR DESCRIPTION
**Pull request by deathbearbrown**
### Description of the problem this pull request fixes

Adds a Line mark to the scene on initialization. 

Changes proposed in this pull request:
- Adds the line mark
- moves the map function used in manipulators into a utility 
- add tests for line file
- add test for property inspector
- add test for map-manipulatory utility
- removes helloworld test because it's dumb
- linting on the tests (that's why so many files have changed >.<)
#### This PR includes:
- [x] Tests
- [x] functional comments
- [x] high level description of any new components

To test:
Select the line,  drop data onto the axis. Change the stroke & color
line

NOTE:  I don't have a test for the manipulator :/   It kept breaking when initalizing even though I was passing it a Vega Model.  Will talk to yall later about that.

### CODE REVIEW

File: src/js/transforms/manipulators/Line.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* eslint no-unused-expressions:0 */
+'use strict';
+var Line = require('./Line.js');
+
+describe('Line Mark Manipulators', function(){
```
Comment (deathbearbrown) [inline src/js/transforms/manipulators/Line.test.js]: working on it now

**Comment by deathbearbrown** [comment]
This is ready for review.

File: src/js/components/inspectors/Line.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  it('renders as a <div>', function() {
+    expect(wrapper.type()).to.eql('div');
+  });
+
+  it('it should render 4 <Property/> components', function() {
+    expect(wrapper.find(Property)).to.have.length(4);
+  });
+
+  it('it should render 4 <Property/> components', function() {
+    expect(wrapper.find('h3')).to.have.length(2);
```
Comment (kadamwhite) [inline src/js/components/inspectors/Line.test.js:41]: Title of this test isn't accurate

File: src/js/components/inspectors/Property.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+var expect = require('chai').expect;
+var React = require('react');
+var enzyme = require('enzyme');
+var Rect = require('../../model/primitives/marks/Rect');
+var Property = require('./Property');
+var wrapper;
+
+describe('Property Inspector <Property/>', function() {
+  describe('Property Inspector <Property/ type="color"> (shallow)', function() {
```
Comment (kadamwhite) [inline src/js/components/inspectors/Property.test.js]: The slash in these describe block names should probably be moved to before the closing `>`

File: src/js/model/primitives/Guide.test.js [hunk 4]
```diff
@@ -15,7 +15,7 @@ beforeEach(function() {
     round: true,
     type: 'ordinal'
   };
-  scale = new Scale(def.name, def.type, undefined, def.range);
+  scale = new Scale(def.name, def.type, null, def.range);
```
Comment (kadamwhite) [inline src/js/model/primitives/Guide.test.js:18]: Aha, clever! That's much nicer

File: src/scss/main.scss [hunk 5]
```diff
@@ -26,7 +26,7 @@ $sidebars-width: $inspector-width + $pipeline-list-width;
 
 $header-font-size: 15px;
 
-@import 'http://idl.cs.washington.edu/static/style/fonts.css';
+// @import 'http://idl.cs.washington.edu/static/style/fonts.css';
```
Comment (kadamwhite) [inline src/scss/main.scss:29]: :+1:

**Comment by kadamwhite** [comment]
This looks great. I had some minor comments around test naming, otherwise once we finish the side conversation about testing wrapper components I'm +1 on this to merge.

File: src/js/components/inspectors/Line.test.js (hunk 2 above)
Comment (deathbearbrown) [inline src/js/components/inspectors/Line.test.js:41]: haha nope, it is not

File: src/scss/main.scss (hunk 5 above)
Comment (deathbearbrown) [inline src/scss/main.scss:29]: omg the console errors were making me nuts

**Pull request merged**